autoscheduler/frontend/src/utils/sortMeetingFunction.tsx

```diff
@@ -0,0 +1,8 @@
+import Meeting from '../types/Meeting';
+
+// sorts meeetings based on meeting type
+const sortMeeting = (a: Meeting, b: Meeting): number => {
```

Comment:
Do these changes need to be in here?

Reply:
nope, he needs to reset to master and remake the changes - this includes the changes from #385